src/compiler/executable.cc

```diff
@@ -44,6 +51,34 @@ static const uint8 VESSEL_TOKEN[] = { VESSEL_TOKEN_VALUES };
 // just much more complicated for something that doesn't change that frequently.
 static int VESSEL_SIZES[] = { 128, 256, 512, 1024, 8192, };
 
+static int sign_if_necessary(const char* out_path) {
+#ifndef TOIT_DARWIN
+  return 0;
+#else
+  char codesign[] = { "codesign" };
```

Comment:
How does one install codesign? Do we need to update a README?

Reply:
I think/hope it's preinstalled. If it isn't I would probably just ship the zsign executable with Toit instead.
This is the approach I would take for cross-compilation too. (That said: currently we ship with non-ARM executables, so the precompiled executables shouldn't complain).